ext/app/electron/logins.ts

```diff
@@ -1,22 +1,29 @@
 import {ApiError} from 'app/common/ApiError';
+import { UserProfile } from 'app/common/LoginSessionAPI';
 import {expressWrap} from 'app/server/lib/expressWrap';
 import {GristLoginMiddleware, GristLoginSystem, GristServer,
         setUserInSession} from 'app/server/lib/GristServer';
-import {getDefaultProfile} from 'app/server/lib/MinimalLogin';
 import {getOrgUrl} from 'app/server/lib/requestUtils';
 import {Request} from 'express';
 
 const cookie = require('cookie');
 
-export type GristElectronAuthMode = 'strict' | 'none' | 'mixed';
+export type GristDesktopAuthMode = 'strict' | 'none' | 'mixed';
+
+export function getProfile(): UserProfile {
+  return {
+    email: process.env.GRIST_DEFAULT_EMAIL as string,
```

Comment:
Can you add a comment explaining why we can rely on these being set, since it isn't obvious locally.

Reply:
Done!